pkg/api/constants.go

```diff
@@ -7,4 +7,7 @@ const (
 
 	// OpenShiftDescription is a common, optional annotation that stores the description for a resource.
 	OpenShiftDescription = "openshift.io/description"
+
+	// OpenShiftInstanceExternalID is a common, optional annotation that stores the OSBAPI instance (UU)ID associated with an object.
+	OpenShiftInstanceExternalID = "openshift.io/instance-external-id"
```

Comment:
"openshift.io" is extremely general and this looks like it is closely related to the openservicebroker api.  If we really believe that this will be the only annotation associated with the openservicebroker API, then we could name it like this, but it seems reasonably likely that we may have more.

Reply:
is there a something.openshift.io that we can/should use instead?  openservicebroker.openshift.io ?

Reply:
i think we get to pick something (and then be told we picked poorly).

openservicebroker.openshift.io/instance-external-id works for me (as would external-instance-id, i'm not sure which is more straightforward.  instance-external-id is somewhat more amenable to expansion in the future i guess (instance-other-property).

Reply:
servicecatalog.openshift.io/instance-external-id ?